src/web/mjs/connectors/VizShonenJump.mjs

```diff
@@ -0,0 +1,200 @@
+import Connector from '../engine/Connector.mjs';
+const EXIF = require('exif-js');
```

Comment:
Pulling in node modules is discouraged as stated in comment above. Use browser version which makes the library available on global scope.